pkg/tfbridge/info.go

```diff
@@ -121,6 +122,33 @@ type ProviderInfo struct {
 	SchemaPostProcessor func(spec *pschema.PackageSpec)
 }
 
+// Send logs or status logs to the user.
+//
+// Logged messages are pre-associated with the resource they are called from.
+type Logger interface {
+	Log
+
+	// Convert to sending ephemeral status logs to the user.
```

Comment:
Nit: if there's any pointers on what these are or how they're different, perhaps a link, that'd be great; but not blocking on this.

Reply:
Opened https://github.com/pulumi/pulumi-terraform-bridge/issues/1338 to track. It's late in the day and I want to get an alpha out today, so I'll come back to this later.